openlibrary/core/models.py

```diff
@@ -285,9 +283,8 @@ def is_in_private_collection(self):
 
     def in_borrowable_collection(self):
         collections = self.get_ia_collections()
-        return ('lendinglibrary' in collections or
-            ('inlibrary' in collections and inlibrary.get_library() is not None)
-            ) and not self.is_in_private_collection()
+        return ('lendinglibrary' in collections
+                and not self.is_in_private_collection())
```

Comment:
This was not ported correctly and is causing lending issues!

Reply:
Missing 'inlibrary' in collections

Reply:
@mekarpeles I'm just seeing this (a closed PR isn't a very visible place to comment). Did you open an issue? Does the problem still exist?

Reply:
Looks like the issue is #3028 which was fixed by PR #3082